github/event_types.go

```diff
@@ -360,6 +361,17 @@ type EditRepo struct {
 	Name *RepoName `json:"name,omitempty"`
 }
 
+// EditOwner represents a change of repository ownership.
+type EditOwner struct {
+	OwnerInfo *OwnerInfo `json:"from,omitempty"`
+}
+
+// OwnerInfo represents the account info of the owner of the repo (could be User or Organization but both are User structs).
+type OwnerInfo struct {
+	Actor *User `json:"user,omitempty"`
```

Comment:
I think the following would be less confusing or at least more consistent.

```suggestion
	User *User `json:"user,omitempty"`
```